pmlc/rt/vulkan/tests/addf.mlir

```diff
@@ -20,7 +20,7 @@ module attributes {
     }
   }
 
-  func @main() {
+  func @main() attributes {entrypoint} {
```

Comment:
We need to resolve this design, which is right now we have two sources of information for the entrypoint. We should decide on having one.

Reply:
+1 -- can we do it in a followup PR, or do you want it resolved in this one?